apps/ui/src/views/Space/Editor.vue

```diff
@@ -316,6 +333,13 @@ function handleFetchPropositionPower() {
   fetchPropositionPower(props.space);
 }
 
+watch(customProposalTime, () => {
+  if (!proposal.value) return;
+
+  proposal.value.start = customProposalTime.start;
+  proposal.value.min_end = customProposalTime.minEnd;
+});
+
```

Comment:
Maybe instead of doing this we just use `proposal.value.start` and `proposal.value.min_end` as storage and avoid having duplicate source of start/end times:
```diff
diff --git a/apps/ui/src/views/Space/Editor.vue b/apps/ui/src/views/Space/Editor.vue
index 54266fd6..8f63547d 100644
--- a/apps/ui/src/views/Space/Editor.vue
+++ b/apps/ui/src/views/Space/Editor.vue
@@ -63,10 +63,6 @@ const modalOpenTerms = ref(false);
 const previewEnabled = ref(false);
 const sending = ref(false);
 const enforcedVoteType = ref<VoteType | null>(null);
-const customProposalTime = reactive<{
-  start?: number;
-  minEnd?: number;
-}>({});
 
 const draftId = computed(() => route.params.key as string);
 const network = computed(() => getNetwork(props.space.network));
@@ -86,6 +82,25 @@ const proposalData = computed(() => {
 
   return JSON.stringify(omit(proposal.value, ['updatedAt']));
 });
+
+type CustomProposalTime = { start?: number; minEnd?: number };
+const customProposalTime = computed({
+  get: () => {
+    if (!proposal.value) return {};
+
+    return {
+      start: proposal.value.start,
+      minEnd: proposal.value.min_end
+    };
+  },
+  set: (value: CustomProposalTime) => {
+    if (!proposal.value) return;
+
+    proposal.value.start = value.start;
+    proposal.value.min_end = value.minEnd;
+  }
+});
+
 const isOffchainSpace = computed(() =>
   offchainNetworks.includes(props.space.network)
 );
@@ -187,15 +202,11 @@ const defaultVotingDelay = computed(() =>
 );
 
 const proposalStart = computed(
-  () =>
-    customProposalTime.start ??
-    proposal.value?.start ??
-    unixTimestamp.value + props.space.voting_delay
+  () => proposal.value?.start ?? unixTimestamp.value + props.space.voting_delay
 );
 
 const proposalMinEnd = computed(
   () =>
-    customProposalTime.minEnd ??
     proposal.value?.min_end ??
     proposalStart.value +
       (props.space.min_voting_period || defaultVotingDelay.value)
@@ -333,13 +344,6 @@ function handleFetchPropositionPower() {
   fetchPropositionPower(props.space);
 }
 
-watch(customProposalTime, () => {
-  if (!proposal.value) return;
-
-  proposal.value.start = customProposalTime.start;
-  proposal.value.min_end = customProposalTime.minEnd;
-});
-
 watch(
   [() => web3.value.account, () => web3.value.authLoading],
   ([toAccount, toAuthLoading], [fromAccount]) => {
```

Reply:
After reading the code again, this `customProposalTime` ref is not needed, as we can can achieve the same result by storing them in the draft directly (this variable was used before I connected the date changes to the draft)